pkg/apis/management.cattle.io/v3/public_authn_types.go

```diff
@@ -14,6 +14,20 @@ type AuthProvider struct {
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
 	Type string `json:"type"`
+
+	// Flag. True when the auth provider supports a `Logout All` operation.
+	// Currently only the SAML providers do, with their `Single Log Out` flow.
+	LogoutAllSupported bool `json:"logoutAllSupported,omitempty"`
+
+	// Flag. True when the auth provider is configured to accept a `Logout All`
+	// operation. Can be set if and only if the provider supports `Logout All`
+	// (see AuthConfig.LogoutAllSupported).
+	LogoutAllEnabled bool `json:"logoutAllEnabled,omitempty"`
+
+	// Flag. Can be set if and only if `LogoutAllEnabled` (above) is set.
```

Comment:
"Can" be set, or "is only meaningful if LogoutAllEnabled is set to true"?

Reply:
Second, is only meaningful ...